# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/d3d12/README.md
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/d3d12/README.md b/third_party/rust/d3d12/README.md
--- a/third_party/rust/d3d12/README.md
+++ b/third_party/rust/d3d12/README.md
@@ -1,6 +1,6 @@
-# d3d12-rs
-[![Crates.io](https://img.shields.io/crates/v/d3d12.svg)](https://crates.io/crates/d3d12)
-[![Docs.rs](https://docs.rs/d3d12/badge.svg)](https://docs.rs/d3d12)
-[![Matrix](https://img.shields.io/matrix/gfx:matrix.org)](https://matrix.to/#/#gfx:matrix.org)
-
-Rust wrapper for raw D3D12 access.
+# d3d12-rs
+[![Crates.io](https://img.shields.io/crates/v/d3d12.svg)](https://crates.io/crates/d3d12)
+[![Docs.rs](https://docs.rs/d3d12/badge.svg)](https://docs.rs/d3d12)
+[![Matrix](https://img.shields.io/matrix/gfx:matrix.org)](https://matrix.to/#/#gfx:matrix.org)
+
+Rust wrapper for raw D3D12 access.